File: engine/model/src/main/scala/tools/packet/mtsoperation/MTSConfirmTransfer.scala
package tools.packet.mtsoperation

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class MTSConfirmTransfer(private var _quantity: Int, private var _position: Int) extends PacketInput {
  def quantity: Int = _quantity

  def position: Int = _position

  override def opcode(): SendOpcode = SendOpcode.MTS_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/mtsoperation/MTSConfirmTransfer.java
package tools.packet.mtsoperation;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record MTSConfirmTransfer(int quantity, int position) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.MTS_OPERATION;
   }
}